generated/niscope/niscope.proto

```diff
@@ -927,7 +909,10 @@ message CheckAttributeViInt32Request {
   nidevice_grpc.Session vi = 1;
   string channel_list = 2;
   NiScopeAttributes attribute_id = 3;
-  sint32 value = 4;
+  oneof value_enum {
```

Comment:
Note: This function signature is kept consistent using override from metadata file. Even though this particular override was not required, since the raw value and enum's constant values are still same (no mappings used yet).